minio/minio
# localizes jstream library and fixes incompatible behaviors

## Description
`mc sql -e="SELECT ...` command for files in `json` and/or in `gz` or `bz2` formats behave differently than amazon s3 version. The error messages thrown was not helpful and incompatible with the s3 counterparts in certain conditions like incorrect `.gz` and `.bz2` files and etc.
Also localized the jstream library.
## Motivation and Context
We need to be compatible with Amazon s3.

## How to test this PR?
Run the following commands with correct and incorrect formatted json, gzipped json and bzip2'ed json files and make sure MinIO AND s3 results match.

```
- mc sql -e="SELECT * from s3object LIMIT 1" --json-input type="document" myminio/test/incorrect.bz2
- mc sql -e="SELECT * from s3object LIMIT 1" --json-input type="document" s3/test/incorrect.bz2
```
## Types of changes

## Checklist:
